docs/team/wpinrui.md

```diff
@@ -4,14 +4,40 @@ title: Ivan's Project Portfolio Page
 ---
 
 ### Project: TutorAid
-
+<!-- PPP format and structure adapted from
+https://github.com/samuelfangjw/tp/blob/master/docs/team/samuelfangjw.md
+-->
 TutorAid is a desktop application used for helping tech-savvy private tutors, who have busy schedules / many students, to keep track of each students' details. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
```

Comment:
Maybe can rephrase the last part of the first sentence as `to keep track of the details of all their students and lessons` so that it better describes the current scope of TutorAid